lib/primitives/block.js

```diff
@@ -839,6 +841,45 @@ class Block extends AbstractBlock {
   static isBlock(obj) {
     return obj instanceof Block;
   }
+
+  /*
+   * Get compact block filter (BIP158)
+   * @param {CoinView} view
+   * @returns {Object} See {@link Golomb}
+   */
```

Comment:
Nit: normally a newline between the docstring and the method definition

Reply:
OK. Added.